crates/ruff/src/noqa.rs

```diff
@@ -105,11 +104,11 @@ fn parse_file_exemption(line: &str) -> ParsedExemption {
         if remainder.is_empty() {
             return ParsedExemption::All;
         } else if let Some(codes) = remainder.strip_prefix(':') {
-            let codes: Vec<&str> = SPLIT_COMMA_REGEX
-                .split(codes.trim())
+            let codes = codes
+                .split(|c: char| c.is_whitespace() || c == ',')
```

Comment:
Our use of is_whitespace throughout the code base is probably incorrect because it, to my knowledge, includes Unicode whitespace that the python syntax does not support